website/docs/r/data_export_rule.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+subcategory: "Data Export Rules"
+layout: "lacework"
+page_title: "Lacework: lacework_data_export_rule"
+description: |-
+  Create and manage Lacework Data Export Rules
+---
+
+# lacework\_data\_export\_rule
+
+Use this resource to export data collected from your Lacework account.
+For more information, see the [Data Export Rules documentation](https://docs.lacework.com/console/category/data-shares--export).
+
```

Comment:
Can we also link this doc: https://docs.lacework.com/console/s3-data-export

We will need to update it 👆🏽 with this new resource.